.azure-pipelines/steps/run-in-docker.yml

```
@@ -37,6 +37,28 @@ steps:
         --env TestAllPackageVersions=$(TestAllPackageVersions) \
         --env IncludeMinorPackageVersions=$(IncludeMinorPackageVersions) \
         --env NUGET_ENABLE_EXPERIMENTAL_HTTP_RETRY=true \
+        --env DD_API_KEY=$(DD_API_KEY) \
```

IIRC, this format _requires_ that `DD_API_KEY` is present, but we might not always want that. I think if you do the following, it only adds it if it's available, and doesn't error if it isn't:

```suggestion
        --env DD_API_KEY \
```